### DISCUSSION TITLE
Allow Japanese communication in this repository(日本語でのやりとりもOKにしませんか？)

### DISCUSSION THREAD

**Discussion by kachick** [mobu-of-the-world/mobu#48] [General]
ユーザーに現時点では日本人が多いと思うので、日本語コミュニケーションも許可してみるのってどうでしょう？
というのと、そうするならするで、 README あたりに書いておいたほうが良いかも？

以下のどれに関して許可するか

* issue
* PR description
* discussions
* comment
* code

あたり、検討する必要はありそうですね。

**Comment by pankona** [comment]
ナイスイシュー！

**Reply by highwide** [reply]
You wrote "nice issue"?

**Comment by highwide** [comment]
Hi, I'm a Japanese user.
But I hope this product would be used all over the world.
So is it really okay to allow to use Japanese?

**Comment by pankona** [comment]
mobu is transfered to mobu-of-the-world organization. We need to make it to be the mobu of the world.
Let's keep to use english in this repo. Thanks of the world.

**Comment by pankona** [comment]
I'm not sure how to make this thread "answered" ...

**Reply by kachick** [reply]
https://docs.github.com/ja/discussions/managing-discussions-for-your-community/moderating-discussions 👀

**Comment by kachick** [comment]
https://github.com/mobu-of-the-world/mobu/issues/292 founder is a rule breaker 👁️
